charts/sonarqube/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ All changes to this chart will be documented in this file.
 * Run the initSysctl init-container as root to prevent 'permission denied' issues
 * Add revisionHistoryLimit configuration for SonarQube application Deployment ReplicaSets & StatefulSets
 * Update the security contexts to use root as group ID
+* Add support for IPv6 single stack clusters
```

Comment:
This line can be removed ?

Reply:
good catch, I updated the changelog accordingly